README.md

```diff
@@ -41,6 +41,117 @@ the last few lessons.
 
 Put your code in `index.js`.
 
+While you will want to be guided by the tests, you will implement the following
+functions. To make the tests easier to read, we've provided the _signatures_ of
+the functions.
+
+A function _signature_ is the function name, the arguments it expects, and what
+the function returns.
+
+### `createEmployeeRecord`
+
+* **Argument(s)**
+  * 4-element Array of a `String`, `String`, `String`, and `Number`
+    corresponding to a first name, family name, title, and pay rate per hour
+* **Returns**
+  * JavaScript `Object` with keys:
+    * `firstName`
+    * `familyName`
+    * `title`
+    * `payPerHour`
+    * `timeInEvents`
+    * `timeOutEvents`
+* **Behavior**
+  * Loads `Array` elements into `Object` and initializes Arrays for storing
+    time in events and time out events
+
+### `createEmployees`
+
+* **Argument(s)**
+  * `Array` of `Arrays`
+* **Returns**
+  * `Array` of `Object`s
+* **Behavior**
+  * Converts each nested `Array` into an employee record using
+    `createEmployeeRecord` and accumulates it to a new `Array`
+
+### createTimeInEvent
+
+* **Argument(s)**
+  * A date stamp
+* **Returns**
+  * The record used as execution context
```

Comment:
This is pretty tricky - should we change this so that it's supposed to return `undefined` instead of `this`?